indexer/road_shields_parser.cpp

```diff
@@ -72,6 +72,9 @@ std::unordered_map<std::string, RoadShieldType> const kRoadNetworkShields = {
     {"pl:national", RoadShieldType::Generic_Red},
     {"pl:regional", RoadShieldType::Generic_Orange},
     {"pl:local", RoadShieldType::Generic_White},
+    {"tr:motorway", RoadShieldType::Generic_Orange},
+    {"tr:road", RoadShieldType::Generic_Blue},
+    {"tr:provincial", RoadShieldType::Generic_White},
```

Comment:
Where are these come from?
When I make the overpass query like "network"~"tr:" (https://overpass-turbo.eu/s/1FVM), I got nothing.

Reply:
I didn't realize that it was case-sensitive and thought it wasn't strictly matching. Here are example relations:
[TR:motorway](https://www.openstreetmap.org/relation/153991)
[TR-roads](https://www.openstreetmap.org/relation/1720464)
[TR:provincial - only one available as a relation](https://www.openstreetmap.org/relation/16142889)

Reply:
Yes, the processing is case sensitive.
Fix TR and dash and add unit tests here: road_shields_parser_tests.cpp

Reply:
Honestly, after thinking about it a bit more, the implementation can wait; because the data on OpenStreetMap is very inconsistent. There are many problems lying with motorway/state road relations, namely:

- some motorway relations include their motorway_link junctions.
- some portions of state roads are mapped with subsection number ([example](https://www.openstreetmap.org/way/191189583)) on their ref tag.
- there is no consistency with the variants of "D".
- and finally, motorway sections on construction are included inside Turkish motorway road network relation or existing motorway relations, in turn which makes the situation worse.

I'd like to consult with the local community and take at least some action towards standardizing reference and relation tags, before going further with this pull request.

Reply:
Ok, remove it for now.